examples/plot_multi_axes.py

```diff
@@ -0,0 +1,113 @@
+"""
+Test Plots with Multiple Axes (Created Using Matplotlib) in Matplotcheck
```

Comment:
@jlpalomino @nkorinek great point about axes vs axis
I actually think that the correct language her is a figure with subplots. 
Yes technically they are matplotlib.axes or whatever the full object name is. And you might call a single "axes" and axis but an axis is also x-axis / y-axis. 
When you setup a plot you name the individual subplots (or axes objects) ax1, ax2 etc. but you set it up as subplots which is a term you'd search on for help so i suggest using that language instead to avoid the confusion.

Reply:
again the matplotlib language is figure, subplots within a figure. ... think `plt.subplots()` that is what actually sets up your subplots.